changelog/59843.fixed

```diff
@@ -0,0 +1,3 @@
+`gpg.absent` ignores parameters when checking keychain.
+`gpg.present` can report wrong key in comments.
+
```

Comment:
This file should have a `.md` file extension
The text should be more clear that these issues are fixed. This is what will be displayed in the changelog.